omnibus/config/projects/push-jobs-client.rb

```diff
@@ -38,15 +38,12 @@
 override :bundler,        version: "1.7.12"
 # Uncomment to pin the chef version
 #override :chef,           version: "12.2.1"
-override :ruby,           version: "2.1.6"
-######
-# Ruby 2.1/2.2 has an error on Windows - HTTPS gem downloads aren't working
-# https://bugs.ruby-lang.org/issues/11033
-# Going to leave 2.1.5 for now since there is a workaround
-override :'ruby-windows', version: "2.1.5"
-override :'ruby-windows-devkit', version: "4.7.2-20130224-1151"
-#override :'ruby-windows', version: "2.0.0-p451"
-######
+if windows?
```

Comment:
This is the core change needed.  The default version for :ruby is now "ruby-windows" which tells it to go with what ruby-windows wants instead of compiling ruby from scratch.  So it's imperative to not override the version of ruby to "2.1.6" on the windows builds.